$NetBSD: patch-configure,v 1.8 2018/02/02 10:03:05 jperkin Exp $

Regen after excluding -latomic test on NetBSD.
Leave pkgsrc to handle security features.

--- configure.orig	2017-11-30 09:43:31.000000000 +0000
+++ configure
@@ -5569,9 +5569,14 @@ else
 fi
 
 
-  case "$host" in
-  mips* | powerpc-* )
-    { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether the linker accepts -latomic" >&5
+  case "$host_os" in
+  netbsd*)
+    : # no -latomic
+    ;;
+  *)
+    case "$host" in
+    mips* | powerpc-* )
+      { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether the linker accepts -latomic" >&5
 $as_echo_n "checking whether the linker accepts -latomic... " >&6; }
     LDFLAGS="-latomic $LDFLAGS"
     cat confdefs.h - <<_ACEOF >conftest.$ac_ext
@@ -5593,7 +5598,9 @@ else
 
 fi
 rm -f core conftest.err conftest.$ac_objext \
-    conftest$ac_exeext conftest.$ac_ext
+      conftest$ac_exeext conftest.$ac_ext
+      ;;
+    esac
     ;;
   esac
 
@@ -6797,7 +6804,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $gl_cv_warn_cxx__fstack_protector" >&5
 $as_echo "$gl_cv_warn_cxx__fstack_protector" >&6; }
-if test "x$gl_cv_warn_cxx__fstack_protector" = xyes; then :
+if test "x$gl_cv_warn_cxx__fstack_protector" = xDISABLED; then :
 
         CFLAGS="-fstack-protector $CFLAGS"
         CXXFLAGS="-fstack-protector $CXXFLAGS"
@@ -20586,7 +20593,7 @@ fi
 
   if test "x$PG_CONFIG" != "x" -o "x$PGSQL_LIBS" = "x"; then :
 
-        if test "x$PG_CONFIG" == "x"; then :
+        if test "x$PG_CONFIG" = "x"; then :
 
       # Extract the first word of "pg_config", so it can be a program name with args.
 set dummy pg_config; ac_word=$2
@@ -20630,7 +20637,7 @@ fi
 
 
 fi
-    if test "x$PG_CONFIG" == "x"; then :
+    if test "x$PG_CONFIG" = "x"; then :
 
       as_fn_error $? "Can not find pg_config, use --with-pg-config to specify the path to pg_config" "$LINENO" 5
 
